### ISSUE TITLE
Question of the paper

### ISSUE THREAD

**Issue by Hugo-cell111** [qpc1611094/FPL#4]
Hi! In the conclusion section you have mentioned "There are still directions worth continuing to explore in FPL, e.g., "extending discrete K values to continuous form for finer-grained fuzzy positive labels", and I wonder what "finer-grained labels" means. Does the continuous label exist? Thanks!

**Comment by qpc1611094** [comment]
In classical classification tasks, the labels are defined as discrete variables, which are instantiated as the weight in classifier layer, and the classification logits are dot products between the image feature vector and these weight vectors.
Thus, one can build new pseudo classification vectors by interpolating existing discrete vectors.
I think this could be seen as 'continuous' label since the interpolation weight is continuous.

**Comment by Hugo-cell111** [comment]
Yes, but I am still difficult to understand it. Could you use some mathmatical definitions or formulas to explain it? Thanks a lot!
